### PR TITLE
Feature/csv export

### PR DESCRIPTION
**Pull request by Switcherlapp97**
# *Name of the PR*

## :recycle: Current situation
*Describe the current situation (if possible with and exemplary (or real) code snippet and/or where this is used)*

## :bulb: Proposed solution
*Describe the solution (if possible with and exemplary (or real) code snippet)*

### Problem that is solved
*Provide a description and link issues that are solved*

### Implications
*Describe the implications, e.g. refactoring*

## :heavy_plus_sign: Additional Information
*Provide some additional information if possible*

### Related PRs
*Reference the related PRs*

### Testing
*Are there tests included? If yes, which situations are tested and which corner cases are missing?*

### Reviewer Nudging
*Where should the reviewer start, where is a good entry point?*

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,111 +1,183 @@
...
-  color: theme.palette.text.secondary,
-  height: "400px",
-  justifyContent: "center",
-  alignItems: "center",
+import { BrowserRouter as Router, Route, Switch, Redirect, Link } from "react-router-dom"
+import routes from "./routes";
+import { Breadcrumbs } from "./features/breadcrumbs/Breadcrumbs";
+import { useSelector } from "react-redux";
+
+import * as React from 'react';
```
Comment (leanderpfeiffer) [inline frontend/src/App.js]: Please do not do this. This isnt nice

**Review by leanderpfeiffer — APPROVED** [review]
